docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-slurm/slurm-requirements.rst

```diff
@@ -116,6 +116,14 @@ to optimize how Determined interacts with PBS:
    other configurations such as targeting a specific resource pool with only GPU nodes, or
    specifying a PBS constraint in the experiment configuration.
 
+   PBS should be configured to provide the environment variable ``CUDA_VISIBLE_DEVICES``
+   (``ROCR_VISIBLE_DEVICES`` for ROCm) to the job based upon the GPUs allocated. If PBS is not
+   configured to provide ``CUDA_VISIBLE_DEVICES``, Determined will utilize a single GPU on each
+   node. To fully utilize mutliple GPUs, you must either manually define ``CUDA_VISIBLE_DEVICES`` or
+   provide the ``pbs.slots_per_node`` setting in your experiment configuration to indicate how many
+   GPU slots are available for use. The format of ``CUDA_VISIBLE_DEVICES`` is an array of GPU IDs
```

Comment:
Perhaps there should be a sentence, before "The format ...",  that says,

For example, if ``pbs.slots_per_node`` is set to 4, then CUDA_VISIBLE_DEVICES will be automatically set to "0,1,2,3" if it hasn't already been set.

Reply:
We never set CUDA_VISIBLE_DEVICES.    

I debated covering this at all, because it really is not our business :-), but I was attempting to provide a hint such that they didn't naively set CUDA_VISIBLE_DEVICES=4 and expect it to mean 0,1,2,3.     Perhaps, I'll just delete it.

Reply:
Ok, I misunderstood what you were trying to say.